data/adapters/dbSNFP_adapter.py

```diff
@@ -0,0 +1,140 @@
+import hashlib
+import os
+import json
+from db.arango_db import ArangoDB
+from adapters import Adapter
+from adapters.helpers import build_variant_id
+
+# Sample file - file has 709 columns:
+# #chr	pos(1-based)	ref	alt	aaref	aaalt	rs_dbSNP	hg19_chr	hg19_pos(1-based)	hg18_chr ... Interpro_domain	GTEx_V8_gene	GTEx_V8_tissue	Geuvadis_eQTL_target_gene
+# Y	2786989	C	A	X	Y	.	Y	2655030	Y	2715030	205	SRY	ENSG00000184895	ENST00000383070	ENSP00000372547 ... . . . . . .
+# Y	2786990	T	C	X	W	.	Y	2655031	Y	2715031	205	SRY	ENSG00000184895	ENST00000383070	ENSP00000372547	... . . . . . .
+
+# Columns of interest:
+# pos  name
+# 0    chr
+# 1    pos(1-based)
+# 2    ref
+# 3    alt
+# 4    aaref: reference amino acid
+# 5    aaalt: alternative amino acid
+# 11   aapos: amino acid position as to the protein.
+# 13   Ensembl_geneid: Ensembl gene id
+# 14   Ensembl_transcriptid
+# 15   Ensembl_proteinid
+# 23   HGVSp_VEP: HGVS protein variant presentation from VEP
+# 29   refcodon: reference codon
+# 30   codonpos: position on the codon (1, 2 or 3)
+# 37   SIFT_score: SIFT score (SIFTori). Scores range from 0 to 1. The smaller the score the
+# 38   SIFT_converted_rankscore: SIFTori scores were first converted to SIFTnew=1-SIFTori,
+# 39   SIFT_pred: If SIFTori is smaller than 0.05 (rankscore>0.39575) the corresponding nsSNV is
+# 40   SIFT4G_score: SIFT 4G score (SIFT4G). Scores range from 0 to 1. The smaller the score the
+# 41   SIFT4G_converted_rankscore: SIFT4G scores were first converted to SIFT4Gnew=1-SIFT4G,
+# 42   SIFT4G_pred: If SIFT4G is < 0.05 the corresponding nsSNV is
+# 695  clinvar_id: clinvar variation ID
+# 696  clinvar_clnsig: clinical significance by clinvar
+# 700  clinvar_var_source: source of the variant
+
+
+class DbSNFPAdapter(Adapter):
+    LABEL = 'dbSNFP_protein_variants'
+
+    SKIP_BIOCYPHER = True
```

Comment:
skipping biocypher so we can use pypy and speed up data loading. This dataset is very big